Userland/Libraries/LibWebView/AriaPropertiesStateModel.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2030, Jonah Shafran <jonahshafran@gmail.com>
```

Comment:
Here as well. Actually, I believe it's just these two files after a quick browser text search.